platform.h

```diff
@@ -65,11 +68,28 @@ typedef struct calvinsys_sensors_environmental_t {
   result_t (*get_temperature)(double *temp);
 } calvinsys_sensors_environmental_t;
 
-void platform_init(struct node_t *node);
+typedef struct platform_t {
```

Comment:
Move this definition to platform_android.c to keep the .h file generic and without ifdefs.